_scripts/instruction-widget/templates/install/debian.html

```diff
@@ -1,3 +1,5 @@
+<p>If you would like to help test a new way to install Certbot that automatically renews your certificates and stays up-to-date, please choose "snapd" in the system dropdown above.</p>
```

Comment:
We don't want these instructions shown for Devuan users. To fix this, I think you can wrap this text with
```
{{^devuan}}
{{/devuan}}
```
but it'd be good to double check this works.

Reply:
Looks like it's working to me.